src/chunk.c

```diff
@@ -1911,7 +1911,7 @@ chunk_resurrect(const Hypertable *ht, int chunk_id)
 {
 	ScanIterator iterator;
 	Chunk *chunk = NULL;
-	int count = 0;
+	PG_USED_FOR_ASSERTS_ONLY int count = 0;
```

Comment:
This is the new warning reported by clang for builds w/o assertions: the variable is initialized and incremented, but the resulting value is only checked in assertions.